app/controllers/late_policies_controller.rb

```diff
@@ -70,8 +70,7 @@ def create
         flash[:notice] = "The penalty policy was successfully created."
         redirect_to action: 'index'
       rescue StandardError
-        flash[:error] = "The following error occurred while saving the penalty policy: "
```

Comment:
What if some error occurs? This catch block will never show any error?

Reply:
Hello,  @yashad All validation errors have been handled and whenever any error occurs it is displayed on the screen.
![error_msg](https://user-images.githubusercontent.com/32436613/67168767-59832180-f375-11e9-92b3-0bc48f3f4f09.png)